projects/observability/src/pages/apis/service-detail/instrumentation/components/progress-circle.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  styleUrls: ['./progress-circle.component.scss'],
+  selector: 'ht-progress-circle',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="ht-progress-circle">
+      <svg class="progress-circle" width="200px" height="200px" xmlns="http://www.w3.org/2000/svg">
+        <circle
+          class="progress-circle-back"
+          cx="80"
+          cy="80"
+          r="58"
```

Comment:
Do you think we should take `radius` as a `prop` with some default value? 
Given this can be generic, we might have `circle` with different properties.

Reply:
Need to spend some time coming up with sensible formulas for `cx`, `cy`, `height` and `width`. If we change the radius, all those values need to change too.

Reply:
Okay, let's leave it for now then, something we will pick up later in the future.

Reply:
✅  This is done now. Was able to make `radius` a prop, changed above mentioned values to be dynamic.